core/http-auth-aws/src/test/java/software/amazon/awssdk/http/auth/aws/internal/signer/V4RequestSignerTest.java

```diff
@@ -106,15 +98,17 @@ public void sign_withPresigned_addsExpirationParam() {
         V4Context ctx = V4RequestSigner.presigned(getProperties(creds), Duration.ZERO).sign(getRequest());
 
         assertEquals("0", ctx.getSignedRequest().rawQueryParameters().get("X-Amz-Expires").get(0));
-        assertEquals("45d00c9c0eb2c05cfe7e42231a572aef30d96d1afb2a205e27d9c6b89867a865", ctx.getSignature());
+        assertEquals("host;x-amz-archive-description",
+                     ctx.getSignedRequest().rawQueryParameters().get("X-Amz-SignedHeaders").get(0));
+        assertEquals("691f39caa2064fe4fb897976dfb4b09df54749c825a5fcd1e2f0b3fcd1bcc600", ctx.getSignature());
     }
 
     @Test
-    public void sign_withNoChecksumHeader_throws() {
+    public void sign_withNoContentHashHeader_throws() {
```

Comment:
I think we'll debate whether this header should always be included or not, but either ways, this test is now sign_withHeader_withNoContentHashHeader_throws()